crates/nu-command/Cargo.toml

```diff
@@ -91,6 +91,7 @@ sqlparser = { version = "0.16.0", features = ["serde"], optional = true }
 [target.'cfg(unix)'.dependencies]
 umask = "2.0.0"
 users = "0.11.0"
+nix = { version = "0.24", default-features = false, features = ["signal"] }
```

Comment:
We're already using the `signal-hook` crate for something similar. I wonder if it can be used here in place of `nix` so we don't add another dependency?

Reply:
Actually `nix` is pulled by `ctrlc`. I'll have a try later.

Reply:
thanks for updating.